tajo-common/src/main/java/org/apache/tajo/util/Pair.java

```diff
@@ -24,11 +24,20 @@
   private T1 first;
   private T2 second;
 
+  public Pair() {
```

Comment:
It is only used for JoinGraphContext::getCachedOrNewJoinEdge. It would be better if JoinGraphContext::getCachedOrNewJoinEdge creates a new instance for Pair instead of reusing a member variable.